LDAP/Init.hsc

```diff
@@ -106,6 +107,15 @@ ldapSimpleBind ld dn passwd =
            return ()
                          )))
 
+{- | Bind with SASL external mechanism. -}
```

Comment:
The two basic bind types are _simple_ and _SASL_. We're indeed implementing the simplest possible SASL bind here, but I'd still rather avoid using the _simple_ term in its name because it mixes unrelated concepts. Since generalizing the function to other SASL bind operations would require substantial effort, we're probably stuck with finding a better name for now. It's ugly and I hate it, but `ldapTrivialExternalSaslBind` is my best suggestion at the moment. It's trivial in the sense that you can't add LDAP controls or your own interaction function.